homeassistant/components/plex/update.py

```diff
@@ -0,0 +1,76 @@
+"""Representation of Plex updates."""
+import logging
+from typing import Any
+
+from plexapi.exceptions import PlexApiException
+import plexapi.server
+import requests.exceptions
+
+from homeassistant.components.update import UpdateEntity, UpdateEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CONF_SERVER_IDENTIFIER
+from .helpers import get_plex_server
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Plex media_player from a config entry."""
```

Comment:
Update entities

Reply:
Thanks. Fixed in https://github.com/home-assistant/core/pull/101815.